server/lib/data_import/my_anime_list/extractor/character.rb

```diff
@@ -0,0 +1,95 @@
+module DataImport
+  class MyAnimeList
+    module Extractor
+      class Character
+        include Enumerable
+
+        attr_reader :dom, :external_id
+
+        def initialize(html, external_id)
+          @dom = Nokogiri::HTML(html)
+          @external_id = external_id
+        end
+
+        def name
+          # line 473 in guts-character.html
+          options = ['Animeography', 'Mangaography', 'Voice Actors']
+          dom.css('.normal_header').map do |row|
+            next if  options.include?(row.content.strip)
+
+            english = row.children.first.content.strip
+            # japanese = row.children.at_css('small').content.strip
+
+            return english
+          end
+        end
+
+        def image
+          # line 319 in guts-character.html
+          dom.at_css('#content > table > tr > td:first-child div a img')
+             .attr('src')
+        end
+
+        def description
+          result = dom.css('#content > table >tr > td:nth-child(2)')
+                      .children.take_while { |x|
+            !x.text.include? 'Voice Actor'
+          }.reject { |x|
```

Comment:
I literally don't know what this wants... It looks apeshit ugly with this indentation. and is already breaking the chaining rule...